### ISSUE TITLE
Issue in writing ascii output

### ISSUE THREAD

**Issue by palgunadi1993** [SeisSol/SeisSol#134]
Hi,

I want to output ascii files for on-fault receivers. The fault is dipping 60degree with strike N214E. I extracted the coordinate of on-fault coordinate using 'Hovers point on' in Paraview. I am able to produce the file accordingly but it raised errors as follow:

srun: error: nid00301: tasks 0-1: Floating point exception
srun: Terminating job step 11277632.0
srun: error: nid00302: task 2: Terminated
srun: error: nid00305: task 8: Terminated
srun: error: nid00304: task 6: Terminated
srun: error: nid00306: tasks 10-11: Terminated
srun: error: nid00303: tasks 4-5: Terminated
srun: error: nid00302: task 3: Terminated
srun: error: nid00305: task 9: Terminated
srun: error: nid00304: task 7: Terminated
srun: Force Terminated job step 11277632.0

the last printout of the progress:
Rank:        0 | Info    | <--------------------------------------------------------->
Rank:        0 | Info    | <     Start calc_SeisSol ...                              >
Rank:        0 | Info    | <--------------------------------------------------------->
Sun Oct 06 17:03:50, Info:  Waiting for last wave field.
Sun Oct 06 17:03:50, Info:  Writing wave field at time 0.
Sun Oct 06 17:03:50, Info:  Writing wave field at time 0. Done.
Sun Oct 06 17:03:50, Info:  Writing free surface at time 0.
Sun Oct 06 17:03:50, Info:  Writing free surface at time 0. Done.
Sun Oct 06 17:03:50, Info:  Writing faultoutput at time 0.
Sun Oct 06 17:03:50, Info:  Writing faultoutput at time 0. Done.
Sun Oct 06 17:03:50, Info:  #max-updates since sync:  0  @  0
Sun Oct  6 17:03:52 +03 2019

My parameters:
&Pickpoint
printtimeinterval = 0.05       ! Index of printed info at timesteps
OutputMask = 1 1 1 1 1   ! output 1/ yes, 0/ no - position: 1/ slip rate 2/ stress 3/ normal velocity 4/ in case of rate and state output friction and state variable
                           ! 5/ background values
nOutpoints = 2
PPFileName = 'on-receivers2.dat'
/

I wonder what makes this error? is it related to the wrong point coordinate or it is related to SeisSol output on ascii file?

Thank you.

Best,
Kadek

**Comment by Thomas-Ulrich** [comment]
Hi,
Please share a full setup so we can investigate your problem.
Thomas.

**Comment by palgunadi1993** [comment]
Hi,

Here is parameters.par:
&equations
MaterialFileName = 'Korea_material_dip.yaml'
Plasticity=1
Tv=0.05                          ! approx: dx/(N+1)/V_s
/

&IniCondition
/

&Boundaries
BC_fs = 1
BC_dr = 1                               ! Fault boundaries
BC_of = 1                               ! Absorbing boundaries
/

&DynamicRupture
FL = 103                    ! Friction law (0: none, 1:self-similar crack, 2:LSW, 3:RS (aging), 4:RS (slip))
BackgroundType = 0         ! Type of background model
ModelFileName = 'Korea_fault_dip.yaml'

RS_f0 = 0.6
RS_sr0 = 1d-6
RS_b = 0.014
Mu_W=0.1
RS_iniSlipRate1 = 1d-16
RS_iniSlipRate2 = 0d0

t_0 = 0.5

GPwise = 1                  ! elementwise =0 ; GPwise =1
XRef = -0.1                  ! Reference point
YRef = 0.0
ZRef = -1.0
refPointMethod = 1

RF_output_on = 0            ! RF on      
magnitude_output_on =0
energy_rate_output_on = 0
OutputPointType = 5         ! Type (0: no output, 1: take GP's 2: 4 points per surface triangle, 3: output at certain pickpoints)
/

&Elementwise
printIntervalCriterion = 2       ! 1=iteration, 2=time
printtimeinterval_sec = 0.05      ! Time interval at which output will be written
OutputMask = 1 1 1 1 1 1 1 1 1 1 1   ! output 1/ yes, 0/ no - position: 1/ slip rate 2/ stress 3/ normal velocity 4/ in case of rate and state output friction and state variable
refinement_strategy = 2
refinement = 1

/
&Pickpoint
printtimeinterval = 0.05       ! Index of printed info at timesteps
OutputMask = 1 1 1 1 1   ! output 1/ yes, 0/ no - position: 1/ slip rate 2/ stress 3/ normal velocity 4/ in case of rate and state output friction and state variable
                           ! 5/ background values
nOutpoints = 2
PPFileName = 'on-receivers2.dat'
/


&SourceType
/

&SpongeLayer
/

&MeshNml
MeshFile = 'mesh/pohang2planes'
meshgenerator = 'PUML'       ! Name of meshgenerator (format)
/


&Discretization
CFL = 0.5                           ! CFL number (<=1.0)
!CFL = 0.2                            ! CFL number (<=1.0)
FixTimeStep = 5                     ! Manualy chosen minimum time
ClusteredLTS = 2
/

&Output
xdmfWriterBackend='posix'
OutputFile = 'output-pohang2planes/Pohang'
!             |stress     |vel  |plast
iOutputMask = 0 0 0 0 0 0 1 1 1 0 0 0 0 0 0 0
Format = 6                           ! Format (0=IDL, 1=TECPLOT, 2=IBM DX, 4=GiD, 6=hdf5, 10= no output))
TimeInterval = 1.                   ! Index of printed info at time
!OutputRegionBounds = 6.10e6 6.27e6 -3.97e6 -3.80e6 -60e3 10e3
printIntervalCriterion = 2
SurfaceOutput = 1
SurfaceOutputRefinement = 1
SurfaceOutputInterval = 1.0
!nRecordPoints = 8
!RFileName = 'rec_pohang2planes.dat'
refinement = 1
pickdt = 0.1                       ! Pickpoint Sampling
pickDtType = 1                      ! Pickpoint Type
/

            

&AbortCriteria
EndTime = 5.0
/

&Analysis
/

&Debugging

no-receivers2.dat (based on 'hover point on' tool in paraview):
-247.862 220.111 -4839.1
481.217 713.02 -4269.6

Fault yaml:
!Switch
[rs_a]: !LayeredModel
  map: !AffineMap
    matrix:
      z0: [0.0, 0.0, 1.0]
    translation:
      z0: 0
  interpolation: linear
  parameters: [rs_a]
  nodes:
     10e10:    [0.02]
     1500.0:  [0.01]
     -4000.0: [0.01]
     -10e10:   [0.01]
[RS_sl0]: !ConstantMap
  map:
    RS_sl0: 0.4
[rs_srW]: !LayeredModel
      map: !AffineMap
        matrix:
          z0: [0.0, 0.0, 1.0]
        translation:
          z0: 0
      interpolation: linear
      parameters: [rs_srW]
      nodes:
         10e10:    [1.0]
         1500.0:  [1.0]
         -4000.0: [0.1]
         -10e10:   [0.1]
[s_xx, s_yy, s_zz, s_xy, s_yz, s_xz]: !Include Korea_initial_stress_dip.yaml
[nuc_xx,nuc_yy,nuc_zz,nuc_xy, nuc_yz,nuc_xz]: !Include Korea_nucleation_stress_dip.yaml

initial stress yaml:
!EvalModel
parameters: [Omega, eCS, b_xx, b_yy, b_zz, b_xy, b_yz, b_xz]
model: !Switch
  #This does not effect the calculations
  [Omega]: !FunctionMap
    map:
      Omega: |
       zStressDecreaseStart = -11000.;
       zStressDecreaseStop = -15000.;
       zStressDecreaseWidth = zStressDecreaseStart - zStressDecreaseStop;

       if  (z>=zStressDecreaseStart) {
          return 1.0;
       } else {
          if (z>=zStressDecreaseStop) {
             a = 1.0-(z-zStressDecreaseStop)/zStressDecreaseWidth;
             Sx = (3.0*a*a-2.0*a*a*a);
             return 1.0-Sx;
          } else {
            return 0.001;
          }
       }
  [b_xx, b_yy, b_zz, b_xy, b_yz, b_xz]: !EvalModel
    parameters: [sig_zz,S]
    model: !FunctionMap
      map:
        sig_zz: |
          return 2670.0*0.15*9.8*min(-1500.0,z);
        S: |
          R = 0.65;
          return (1.0/R-1.0);
    components: !AndersonianStress
      constants:
        mu_d:      0.1
        mu_s:      0.6
        #strike alpha means Shmax=alpha+30
        SH_max:   130.0
        cohesion:  0.0
        #s2ratio<0.5 induces transpression, else transtension
        s2ratio:   0.1
        S_v: 3
  [eCS]: !FunctionMap
    map:
      #avoid positive stress above z=0
      #0.45 used with 125
      eCS: |
        return 2670.0*0.15*9.8*min(-1500.0,z);
components: !FunctionMap
  map:
    s_xx:     return Omega*b_xx + (1.0-Omega)*eCS;
    s_yy:     return Omega*b_yy + (1.0-Omega)*eCS;
    s_zz:     return Omega*b_zz + (1.0-Omega)*eCS;
    s_xy:     return Omega*b_xy;
    s_yz:     return Omega*b_yz;
    s_xz:     return Omega*b_xz;

nucleation:
!EvalModel
parameters: [b_xx, b_yy, b_zz, b_xy, b_yz, b_xz,ShapeNucleation]
model: !Switch
  [b_xx, b_yy, b_zz, b_xy, b_yz, b_xz]: !EvalModel
    parameters: [S, sig_zz]
    model: !FunctionMap
      map:
        sig_zz: |
          return -(2670.0-1000.0)*9.8*5000.;
        S: |
          R = 1.8;
          return (1.0/R-1.0);
    components: !AndersonianStress
      constants:
        mu_d:      0.1
        mu_s:      0.6
        #strike alpha means Shmax=alpha+30
        SH_max:   77.0
        cohesion:  0.0
        #s2ratio<0.5 induces transpression, else transtension
        s2ratio:   0.1
        S_v: 3
  [ShapeNucleation]: !Any
    components: !FunctionMap
      map:
        ShapeNucleation: |
          xc =  0;
          yc =  0;
          zc = -4270;
          r_crit = 580.0;
          r = sqrt(pow(x-xc, 2.0) + pow(y-yc, 2.0) + pow(z-zc, 2.0));
          if (r < r_crit) {
            return exp(pow(r,2.0)/(pow(r,2.0)-pow(r_crit,2.0)));
          }
          return 0.0;
components: !FunctionMap
  map:
    nuc_xx:     return ShapeNucleation*b_xx;
    nuc_yy:     return ShapeNucleation*b_yy;
    nuc_zz:     return ShapeNucleation*b_zz;
    nuc_xy:     return ShapeNucleation*b_xy;
    nuc_yz:     return ShapeNucleation*b_yz;
    nuc_xz:     return ShapeNucleation*b_xz;

Material:
!Switch
[rho, mu, lambda]:  !LayeredModel
  map: !AffineMap
    matrix:
      z1: [0.0, 0.0, 1.0]
    translation:
      z1: 0
  interpolation: lower
  parameters: [rho, mu, lambda]
  nodes:
    0.000000: [1930.000000,4.44672e+08,4.49323e+09]
    -203.000000: [1930.000000,4.44672e+08,4.49323e+09]
    -203.000010: [1930.000000,9.42631e+09,1.2182e+10]
    -670.000000: [1930.000000,9.42631e+09,1.2182e+10]
    -670.000010: [2100.000000,1.92799e+10,1.56337e+10]
    -2400.000000: [2100.000000,1.92799e+10,1.56337e+10]
    -2400.000010: [2270.000000,2.13945e+10,2.46356e+10]
    -3400.000000: [2270.000000,2.13945e+10,2.46356e+10]
    -3400.000010: [2420.000000,2.65138e+10,2.97909e+10]
    -7700.000000: [2420.000000,2.65138e+10,2.97909e+10]
    -7700.000010: [2730.000000,3.36339e+10,2.8086e+10]
    -12000.000000: [2730.000000,3.36339e+10,2.8086e+10]
    -12000.000010: [2750.000000,3.76475e+10,3.87574e+10]
    -34000.000000: [2750.000000,3.76475e+10,3.87574e+10]
    -34000.000010: [2750.000000,5.819e+10,6.18269e+10]
    -.inf: [2750.000000,5.819e+10,6.18269e+10]
[bulkFriction]: !ConstantMap
  map:
    bulkFriction: 0.6
[plastCo]: !IdentityMap
  components:
    - !SphericalDomainFilter
      radius: 1.6e3
      centre:
        x:  0
        y:  0
        z:  -4270
      components: !ConstantMap
        map:
          plastCo: 1.0e20
    - !LayeredModel
      map: !AffineMap
        matrix:
          z0: [0.0, 0.0, 1.0]
        translation:
          z0: 0
      interpolation: linear
      parameters: [plastCo]
      nodes:
        10e10:    [4e6]
        -500.0:  [4e6]
        -3000.0:  [8e6]
        -6000.0:  [5e7]
        -10e10:   [5e7]
[s_xx, s_yy, s_zz, s_xy, s_yz, s_xz]: !Include Korea_initial_stress_dip.yaml

full error file:
mem-bind=LOC - nid00301, task  1  1 [38122]: mask 0x2 set
mem-bind=LOC - nid00301, task  0  0 [38121]: mask 0x1 set
mem-bind=LOC - nid00302, task  2  0 [4096]: mask 0x1 set
mem-bind=LOC - nid00303, task  4  0 [24056]: mask 0x1 set
mem-bind=LOC - nid00304, task  7  1 [23717]: mask 0x2 set
mem-bind=LOC - nid00305, task  9  1 [49076]: mask 0x2 set
mem-bind=LOC - nid00306, task 10  0 [1221]: mask 0x1 set
mem-bind=LOC - nid00303, task  5  1 [24057]: mask 0x2 set
mem-bind=LOC - nid00302, task  3  1 [4097]: mask 0x2 set
mem-bind=LOC - nid00304, task  6  0 [23716]: mask 0x1 set
mem-bind=LOC - nid00305, task  8  0 [49075]: mask 0x1 set
mem-bind=LOC - nid00306, task 11  1 [1222]: mask 0x2 set
Rank:        0 | Warning | Ignoring min space order from parameter file, using           5
srun: error: nid00301: tasks 0-1: Floating point exception
srun: Terminating job step 11277632.0
srun: error: nid00302: task 2: Terminated
srun: error: nid00305: task 8: Terminated
srun: error: nid00304: task 6: Terminated
srun: error: nid00306: tasks 10-11: Terminated
srun: error: nid00303: tasks 4-5: Terminated
srun: error: nid00302: task 3: Terminated
srun: error: nid00305: task 9: Terminated
srun: error: nid00304: task 7: Terminated
srun: Force Terminated job step 11277632.0


output log file:
Sun Oct  6 17:03:09 +03 2019
working directory is
/project/k1343/palgunkh/Pohang
printing modules
setting stripe size
executing SeisSol
Sun Oct 06 17:03:14, Info:  Welcome to SeisSol 
Sun Oct 06 17:03:14, Info:  Copyright (c) 2012-2016, SeisSol Group 
Sun Oct 06 17:03:14, Info:  Built on: Sep 21 2019 13:44:48 (generated kernels) 
Sun Oct 06 17:03:14, Info:  Version: 201807_Kaikoura-350-g5897146 
Sun Oct 06 17:03:14, Info:  Running on: nid00301 
Sun Oct 06 17:03:14, Info:  Using OMP with #threads/rank: 16 
Sun Oct 06 17:03:14, Info:  The stack size ulimit is unlimited. 
Rank:        0 | Info    | <--------------------------------------------------------->
Rank:        0 | Info    | <                SeisSol MPI initialization               >
Rank:        0 | Info    | <--------------------------------------------------------->
Rank:        0 | Info    |  Double precision used for real.
Rank:        0 | Info    |  Setting MPI_AUTODOUBLE to MPI_DOUBLE_PRECISION
Rank:        0 | Info    |  MPI_AUTO_REAL feature initialized. 
Rank:        0 | Info    |  MPI initialization done. 
Rank:        0 | Info    | <--------------------------------------------------------->
 INFORMATION: The assumed unit number is           6 for stdout and           0 
 for stderr.
              If no information follows, please change the value.
Rank:        0 | Info    | <--------------------------------------------------------->
Rank:        0 | Info    | <     Start ini_SeisSol ...                               >
Rank:        0 | Info    | <--------------------------------------------------------->
Rank:        0 | Info    | <  Parameters read from file: parameters-rs.par              >
Rank:        0 | Info    | <                                                         >
Rank:        0 | Info    | (Drucker-Prager) plasticity assumed .
Rank:        0 | Info    | Nodal Basis approach used for plasticity.
Rank:        0 | Info    | Plastic relaxation Tv is set to:   5.000000000000000E-002
Rank:        0 | Info    | High-order points are used for plasticity. 
Rank:        0 | Info    | GPwise initialization. 
Rank:        0 | Info    | Read a PUML mesh file
Rank:        0 | Info    | Output data is in XDMF format (new implementation)
Rank:        0 | Info    | Output data are generated at delta T=    1.00000000000000     
Rank:        0 | Info    | Use POSIX XdmfWriter backend
Rank:        0 | Info    | Refinement strategy is Face Extraction :  4 subcells per cell
Sun Oct 06 17:03:14, Info:  Running mini SeisSol to determine node weight 
Sun Oct 06 17:03:14, Info:  Node weights: mean = 3.74737  std = 0.0361097  min = 3.69636  median = 3.7406  max = 3.80446 
Sun Oct 06 17:03:14, Info:  Reading PUML mesh mesh/pohang2planes 
Sun Oct 06 17:03:14, Info:  Found 5464100 cells 
Sun Oct 06 17:03:15, Info:  Found 995455 vertices 
Sun Oct 06 17:03:23, Info:  Computing LTS weights. 
Sun Oct 06 17:03:28, Info:  Computing LTS weights. Done.  (46163 reductions.)
Sun Oct 06 17:03:42, Info:  Reading mesh. Done. 
Sun Oct 06 17:03:42, Info:  Extracting fault information 
Sun Oct 06 17:03:44, Info:  Mesh initialized in: 30.0054 (min: 30.003, max: 30.0075)
Rank:        0 | Info    | Path to the DG directory is: /project/k1343/palgunkh/myLibs/SeisSol/Maple/
Rank:        0 | Info    | Reading basis functions and mass matrices for DG method 
Rank:        0 | Info    | from file /project/k1343/palgunkh/myLibs/SeisSol/Maple/BasisFunctions2D.tri
Rank:        0 | Info    | Reading basis functions of order            0
Rank:        0 | Info    | Reading mass matrices   of order            0
Rank:        0 | Info    | Reading basis functions of order            1
Rank:        0 | Info    | Reading mass matrices   of order            1
Rank:        0 | Info    | Reading basis functions of order            2
Rank:        0 | Info    | Reading mass matrices   of order            2
Rank:        0 | Info    | Reading basis functions of order            3
Rank:        0 | Info    | Reading mass matrices   of order            3
Rank:        0 | Info    | Reading basis functions of order            4
Rank:        0 | Info    | Reading mass matrices   of order            4
Rank:        0 | Info    | Tetrahedral Elements 
Rank:        0 | Info    | Reading basis functions and mass matrices for DG method 
Rank:        0 | Info    |   from file /project/k1343/palgunkh/myLibs/SeisSol/Maple/BasisFunctions3D.tri
Rank:        0 | Info    | Reading basis functions of order            0
Rank:        0 | Info    | Reading mass matrices   of order            0
Rank:        0 | Info    | Reading basis functions of order            1
Rank:        0 | Info    | Reading mass matrices   of order            1
Rank:        0 | Info    | Reading basis functions of order            2
Rank:        0 | Info    | Reading mass matrices   of order            2
Rank:        0 | Info    | Reading basis functions of order            3
Rank:        0 | Info    | Reading mass matrices   of order            3
Rank:        0 | Info    | Reading basis functions of order            4
Rank:        0 | Info    | Reading mass matrices   of order            4
Sun Oct 06 17:03:46, Info:  Deriving clusters ids for min. time step width / multiRate: 4.83907e-05 / 2 
Sun Oct 06 17:03:46, Info:  Number of elements in time clusters: 
Sun Oct 06 17:03:46, Info:  0: 367 
Sun Oct 06 17:03:46, Info:  1: 8481 
Sun Oct 06 17:03:46, Info:  2: 160983 
Sun Oct 06 17:03:46, Info:  3: 1836353 
Sun Oct 06 17:03:46, Info:  4: 1692447 
Sun Oct 06 17:03:46, Info:  5: 1564466 
Sun Oct 06 17:03:46, Info:  6: 185236 
Sun Oct 06 17:03:46, Info:  7: 15753 
Sun Oct 06 17:03:46, Info:  8: 14 
Sun Oct 06 17:03:46, Info:  maximum theoretical speedup (compared to GTS): 18.0906 per cell LTS, 12.646 with the used clustering. 
Sun Oct 06 17:03:46, Info:  Number of elements in dynamic rupture time clusters: 
Sun Oct 06 17:03:46, Info:  0 (dr): 206 
Sun Oct 06 17:03:46, Info:  1 (dr): 1102 
Sun Oct 06 17:03:46, Info:  2 (dr): 6396 
Sun Oct 06 17:03:46, Info:  3 (dr): 0 
Sun Oct 06 17:03:46, Info:  4 (dr): 0 
Sun Oct 06 17:03:46, Info:  5 (dr): 0 
Sun Oct 06 17:03:46, Info:  6 (dr): 0 
Sun Oct 06 17:03:46, Info:  7 (dr): 0 
Sun Oct 06 17:03:46, Info:  8 (dr): 0 
Rank:        0 | Info    | Synchronizing copy cell material data.
Rank:        0 | Info    | Initializing element local matrices.
Rank:        0 | Info    | DG initial condition projection... 
Rank:        0 | Info    | DG initial condition projection done. 
Rank:        0 | Info    | <--------------------------------------------------------->
Rank:        0 | Info    | <     Start inioutput_SeisSol ...                         >
Rank:        0 | Info    | <--------------------------------------------------------->
Sun Oct 06 17:03:47, Info:  Initializing free surface integrator. 
Sun Oct 06 17:03:47, Info:  Initializing free surface integrator. Done. 
Sun Oct 06 17:03:47, Info:  Wave field writer thread affinity: "----------|------6789|0123456789|0123456789|0123456789|0123456789|0123" 
Sun Oct 06 17:03:47, Info:  Initializing XDMF wave field output. 
Sun Oct 06 17:03:47, Info:  Refinement Startegy is "Divide by 4" 
Sun Oct 06 17:03:47, Info:  Refinement class initialized 
Sun Oct 06 17:03:47, Info:  VariableSubsampler initialized 
Sun Oct 06 17:03:47, Info:  Initialize low order output 
Sun Oct 06 17:03:49, Info:  High order output initialized 
Sun Oct 06 17:03:50, Info:  Low order output initialized 
Sun Oct 06 17:03:50, Info:  Initializing XDMF wave field output. Done. 
Sun Oct 06 17:03:50, Info:  Initializing free surface output. 
Sun Oct 06 17:03:50, Info:  Free surface writer thread affinity: "----------|------6789|0123456789|0123456789|0123456789|0123456789|0123" 
Sun Oct 06 17:03:50, Info:  Finding meshIds for receivers... 
Sun Oct 06 17:03:50, Info:  Cleaning possible double occurring receivers for MPI... 
Sun Oct 06 17:03:50, Info:  Mapping receivers to LTS cells... 
Sun Oct 06 17:03:50, Info:  Initializing XDMF fault output. 
Sun Oct 06 17:03:50, Info:  Fault writer thread affinity: "----------|------6789|0123456789|0123456789|0123456789|0123456789|0123" 
Sun Oct 06 17:03:50, Info:  Initializing free surface output. Done. 
Sun Oct 06 17:03:50, Info:  Initializing XDMF fault output. Done. 
Rank:        0 | Info    | <--------------------------------------------------------->
Rank:        0 | Info    | <     Start calc_SeisSol ...                              >
Rank:        0 | Info    | <--------------------------------------------------------->
Sun Oct 06 17:03:50, Info:  Waiting for last wave field. 
Sun Oct 06 17:03:50, Info:  Writing wave field at time 0.
Sun Oct 06 17:03:50, Info:  Writing wave field at time 0. Done.
Sun Oct 06 17:03:50, Info:  Writing free surface at time 0.
Sun Oct 06 17:03:50, Info:  Writing free surface at time 0. Done.
Sun Oct 06 17:03:50, Info:  Writing faultoutput at time 0.
Sun Oct 06 17:03:50, Info:  Writing faultoutput at time 0. Done.
Sun Oct 06 17:03:50, Info:  #max-updates since sync:  0  @  0 
Sun Oct  6 17:03:52 +03 2019


Kadek

**Comment by Thomas-Ulrich** [comment]
Hi,
So I can reproduce the error.
This is what I got when running in debug mode:

```
forrtl: severe (408): fort: (7): Attempt to use pointer RECPOINT when it is not associated with a target

Image              PC                Routine            Line        Source             
SeisSol_debug_gen  0000000000DFA4A6  Unknown               Unknown  Unknown
SeisSol_debug_gen  0000000000C0B274  ini_faultoutput_m         984  ini_faultoutput.f90
SeisSol_debug_gen  0000000000B915A1  inioutput_seissol         178  inioutput_seissol.f90
SeisSol_debug_gen  0000000000CE7199  fortran_main              207  seissolxx.f90
SeisSol_debug_gen  0000000000D4102B  main                       56  main.cpp
libc-2.22.so       00002ADEC9184725  __libc_start_main     Unknown  Unknown
SeisSol_debug_gen  00000000005C7329  Unknown               Unknown  Unknown
```
any idea?

Thomas.

**Comment by uphoffc** [comment]
I don't know what causes this error but from the lines of code affected it seems to be related to the XDMF output, and not related to the pickpoints.

**Comment by Thomas-Ulrich** [comment]
That 's also what I thought, but when turning off pickpoint output  (```OutputPointType = 4```), the code works fine.

**Comment by palgunadi1993** [comment]
Yes, that's right! when I turned off the pickpoint output it works fine.

**Comment by krenzland** [comment]
See #150.

**Comment by sebwolf-de** [comment]
Is this fixed by PR #185

**Comment by sebwolf-de** [comment]
Guess this was solved...
